src/machines/java/com/enderio/machines/common/blockentity/WiredChargerBlockEntity.java

```diff
@@ -0,0 +1,104 @@
+package com.enderio.machines.common.blockentity;
+
+import com.enderio.api.capacitor.CapacitorModifier;
+import com.enderio.api.capacitor.QuadraticScalable;
+import com.enderio.api.io.energy.EnergyIOMode;
+import com.enderio.core.common.sync.FloatDataSlot;
+import com.enderio.core.common.sync.SyncMode;
+import com.enderio.machines.common.blockentity.base.PoweredMachineEntity;
+import com.enderio.machines.common.io.item.MachineInventoryLayout;
+import com.enderio.machines.common.io.item.SingleSlotAccess;
+import com.enderio.machines.common.menu.WiredChargerMenu;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.block.entity.BlockEntityType;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.common.capabilities.ForgeCapabilities;
+import net.minecraftforge.energy.IEnergyStorage;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Optional;
+
+public class WiredChargerBlockEntity extends PoweredMachineEntity {
+
+    public static final QuadraticScalable CAPACITY = new QuadraticScalable(CapacitorModifier.ENERGY_CAPACITY, () -> 100000f);
+    public static final QuadraticScalable TRANSFER = new QuadraticScalable(CapacitorModifier.ENERGY_TRANSFER, () -> 100000f);
```

Comment:
Energy transfer caps have been removed, you may want to update your branch as this might cause issues :)

Reply:
Okay update to this; looks like you have updated your branch and build is currently failing, so you'll want to just remove anywhere this field is used, and the field itself :)